xcode/Subconscious/Shared/Components/FirstRun/RecoveryPhraseView.swift

```diff
@@ -42,10 +63,61 @@ struct RecoveryPhraseView: View {
     }
 }
 
+enum RecoveryPhraseAction: Hashable {
```

Comment:
Child component has its own actions, model, and update. It is required for the parent component to stitch these into its actions, model, and update.